### PR TITLE
docs: edit user guide and site layout

### PR DESCRIPTION
**Pull request by danielleloh**
a neater and well-organised user guide and site layout will allow for users to navigate through everything easily.

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -880,18 +930,18 @@ Some commands require you to include parameters. These parameters are identified
...
+| p\|    | Phone Number                | 30          | It can start with an optional `+` to indicate the country code, with the rest only containing numbers. It should be at least 3 digits long.                                                                                                                                                                                                                                                                                                                                                                                      |
+| e\|    | Email                       | 50 before and after `@` | **Format:** local-part@domain<br/> **Constraints for local part:**<br/> • Should only contain alphanumeric characters, and the characters `+`, `_`, `.` and `-`<br/> • Should not start with special characters<br/> **Constraints for domain:**<br/> • Made up of domain labels followed by periods<br/> • Must end with a domain label of at least 2 characters long<br/> • Should start and end with alphanumeric characters<br/> • Domain label should consist of alphanumeric characters separated only by hyphens, if any |
```
Comment (Zer0Legion) [inline docs/UserGuide.md:938]: Could the max length for email be updated to 100 in total? I have changed the code such that it minimizes confusion. Thanks!

**Review by Zer0Legion — COMMENTED** [review]
Looks good and is suitable for 1.3 release!

File: docs/UserGuide.md [hunk 2]
```diff
@@ -202,11 +198,14 @@ Imports all persons and their details from a CSV file of your specification.
 
 * The file path should be **absolute**.
 * This command will only import persons' particulars. To import exam scores, take a look at [`importExamScores`](#importing-exam-scores-from-a-csv-file--importexamscores).
+* All compulsory parameters **must** be present in the column headings (i.e. the first row) of the CSV file you are importing i.e. `name`, `email`, `phone` and `address`.
```
Comment (jayllo-c) [inline docs/UserGuide.md:201]: This description is clear and will ensure users understand how to use the import command

File: docs/UserGuide.md [hunk 3]
```diff
@@ -544,12 +566,13 @@ Adds an exam into your exam list.
 
 <box type="info" seamless>
 
-**Important:** Each exam should have a unique name. Avengers Assemble does not allow for exams with duplicate names to be added.
+**Important:** 
 
-</box>
+Each exam should have a unique name. Avengers Assemble does not allow for exams with duplicate names to be added.
```
Comment (jayllo-c) [inline docs/UserGuide.md:571]: Nice addition

File: docs/UserGuide.md [hunk 4]
```diff
@@ -622,6 +647,26 @@ You will see this message once you have successfully selected an exam, including
 
 <br>
 
+<div style="page-break-after: always;"></div>
```
Comment (jayllo-c) [inline docs/UserGuide.md:650]: Good catch ensuring all commands implemented are reflected in the user guide

File: docs/UserGuide.md [hunk 5]
```diff
@@ -724,8 +767,15 @@ Deletes a specified person's exam score.
 
 </box>
 
-**Example:** `deleteScore 1`
+<box type="info" seamless>
+
+**Important:** 
 
+An exam must be selected for this command to work! You can use the [`selectExam`](#selecting-an-exam--selectexam) command to do so.
```
Comment (jayllo-c) [inline docs/UserGuide.md:774]: Nice addition, makes it more straightforward

File: docs/UserGuide.md [hunk 6]
```diff
@@ -880,18 +930,18 @@ Some commands require you to include parameters. These parameters are identified
...
-| \{mt\|\} | More Than                | 7 + 2 decimals | Should be a positive numerical value smaller than the currently selected Exam's Max Score                                                                                                                                                                                                                                                                                                                                                                                                                                        |
+| Prefix | What it refers to          | Max. Length | Constraints    |
+|--------|-----------------------------|-------------|----------------|
```
Comment (jayllo-c) [inline docs/UserGuide.md:934]: Glad this was rectified. In future we need to ensure merges do not revert changes made.

**Review by jayllo-c — COMMENTED** [review]
Nice changes on the UG that makes it more readable

**Pull request merged**